frontend/src/components/BoardUnit.tsx

```diff
@@ -11,6 +10,7 @@ interface Props {
 
 const BoardUnit: React.FunctionComponent<Props> = ({ unit }: Props) => {
   const [hover, setHover] = useState<boolean>(false);
+  const [unitId] = useState<string>(unit.id);
```

Comment:
Could you elaborate how these changes fix the hover being broken?

Reply:
The problem was that
1. There is a unit with id=1 on board, and so with element id `#unit-1`
2. I change units on board
3. The popover is trying to open itself but it has not yet gotten the new state (so it still thinks that it should show itself for the old id), but the DOM is already updated, so `#unit-1` is not present there

When you change this to be the state, the component will re-render in proper order. So first, it gets the new state (from redux), than it changes its internal state, and then it updates itself.

Reply:
This gave me a clue: https://github.com/reactstrap/reactstrap/issues/773